### PR TITLE
data:harisdev

### PR DESCRIPTION
**Pull request by harisdev-netizen**
## Changes proposed

Adding a profile to the LinkFree account

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Thank you, for letting me create a profile on LinkFree.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5860"></a>

### CODE REVIEW

File: data/harisdev.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+    "name": "Haris Khan",
+    "displayStatsPublic": true,
```
Comment (amandamartin-dev) [inline data/harisdev.json:3]:
```suggestion
   
```

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

File: data/harisdev.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "type": "personal",
+    "bio": "Computer Science guy in a Software Engineering world!",
+    "avatar": "https://avatars.githubusercontent.com/u/81189198?v=4.png"
```
Comment (amandamartin-dev) [inline data/harisdev.json:6]:
```suggestion
    "avatar": "https://avatars.githubusercontent.com/u/81189198?v=4.png",
```

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

**Review by amandamartin-dev — DISMISSED** [review]

File: data/harisdev.json (hunk 2 above)
Comment (SaraJaoude) [reply data/harisdev.json]: As mentioned by Amanda, the avatar is no longer required.

**Comment by SaraJaoude** [comment]
Please change your json file so that this matches your GitHub username exactly:

from `data/harisdev.json` to `data/harisdev-netizen.json`

**Comment by harisdev-netizen** [comment]
@SaraJaoude can you please guide me with changing my source json file?

**Comment by SaraJaoude** [comment]
> @SaraJaoude can you please guide me with changing my source json file?

Here are a few screenshots that I hope help:

In `Files Changed` click on the three dots `...` and then `Edit`




Change the current name from data/harisdev.json to `data/harisdev-netizen.json`



Scroll down and commit the changes to `main`

**Comment by harisdev-netizen** [comment]
Thank you @SaraJaoude for the heads up. Did as you meantioned.

File: data/harisdev-netizen.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      "url": "https://github.com/harisdev-netizen"
+    },
+    {
+      "icon": "SiHashnode",
+      "url": "https://harisdev.hashnode.dev"
+    },
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/WittyXEngineer"
+    },
```
Comment (SaraJaoude) [inline data/harisdev-netizen.json:21]:
```suggestion
    }
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by harisdev-netizen** [comment]
@SaraJaoude I have a question. I want to make few changes to my LinkFree, so after I am done doing changes should I create new branch or just commit directory to the main? I hope I'm not bothering you.

**Comment by kumarsonsoff3** [comment]
Hey @harisdev-netizen, it's always recommended to commit the changes into a `new branch` to avoid conflicts. 

For a proper guide on editing your profile, you can check out these docs: https://linkfree.io/docs/how-to-guides/editing

**Comment by harisdev-netizen** [comment]
> Hey @harisdev-netizen, it's always recommended to commit the changes into a `new branch` to avoid conflicts.
> 
> For a proper guide on editing your profile, you can check out these docs: https://linkfree.io/docs/how-to-guides/editing

Understood and followed as mentioned in the docs but I am not able to find the **harisdev-netizen.json** I created. And the LinkFree > data shows **Sorry, we had to truncate this directory to 1,000 files. 763 entries were omitted from the list.**

**Comment by kumarsonsoff3** [comment]
It's because GitHub UI can only show 1000 files in a directory. You can find your File using the button <kbd>Go to file</kbd>  > `Type your filename`. 

Here is the direct link to your file: https://github.com/EddieHubCommunity/LinkFree/blob/main/data/harisdev-netizen.json

**Pull request merged**